src/index.ts

```diff
@@ -267,20 +271,26 @@ jQuery(async function ($) {
               throw new Error('Password is too weak');
             }
 
+            this.echo(`Creating new account...`);
+
             this.pause();
-            await this.account.init(seed, password, isNewAccount, initLibCallback);
+            await this.account.attachAccount(accountName, seed, password, true);
```

Comment:
It doesn't look right to me because I can't restore my account state if I've cleaned the browser store.

Reply:
I guess we should return back isNewAccount and related logic.

Reply:
Fixed in https://github.com/zkBob/zkbob-console/pull/63/commits/f0cb9cddfc1ebb0c0277284a12638156bf009e44